fvm/evm/handler/handler.go

```diff
@@ -558,10 +558,10 @@ func (a *Account) transfer(to types.Address, balance types.Balance) (*types.Resu
 // Deploy deploys a contract to the EVM environment
 // the new deployed contract would be at the returned address and
```

Comment:
I believe this line also needs an update, to describe that we now return a `types.ResultSummary` instead of a `types.Address`.